src/backend/distributed/commands/table.c

```diff
@@ -1824,6 +1895,18 @@ ConstrTypeUsesIndex(ConstrType constrType)
 }
 
 
+/*
+ * ConstrTypeSupportsDefaultNaming returns true if we can generate a default name for the given constraint type
+ */
+bool
+ConstrTypeSupportsDefaultNaming(ConstrType constrType)
```

Comment:
Are these the only types that support default naming in Postgres? If not, this function name is misleading.

Reply:
No, postgres supports much more. Those are the ones we can default name for now.